aten/src/ATen/native/Copy.cpp

```diff
@@ -78,7 +78,7 @@ void copy_same_type_transpose_(Tensor& self, const Tensor& src) {
 // (e.g. XLA) may be supported by overriding copy_ and _copy_from.
 bool is_supported_device(Device device) {
   DeviceType device_type = device.type();
-  return device_type == kCPU || device_type == kCUDA || device_type == kHIP;
+  return device_type == kCPU || device_type == kCUDA || device_type == kHIP || device_type == kFPGA;
```

Comment:
see the XLA comment above -- does that not work?

Reply:
This should work. I will do that.

Reply:
@gchanan,

Thanks. This worked. I only have to modify DispatchStub.h to make things work now.